internal/services/costmanagement/resourceids.go

```diff
@@ -5,3 +5,4 @@ package costmanagement
 //go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=AnomalyAlertView -id=/subscriptions/12345678-1234-9876-4563-123456789012/providers/Microsoft.CostManagement/views/ms:DailyAnomalyByResourceGroup
 //go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=SubscriptionCostManagementView -id=/subscriptions/12345678-1234-9876-4563-123456789012/providers/Microsoft.CostManagement/views/view1
 //go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=ResourceGroupCostManagementView -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.CostManagement/views/view1
+//go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=SubscriptionCostManagementScheduledAction -id=/subscriptions/12345678-1234-9876-4563-123456789012/providers/Microsoft.CostManagement/scheduledActions/scheduledaction1
```

Comment:
From what I can see this isn't actually needed anymore and the generated files in the parse and validate directories can be cleaned up.

Reply:
Done 🥁